### ISSUE TITLE
PHP marked as PAWN

### ISSUE THREAD

**Issue by ItangSanjana** [github/linguist#2972]
Pure PHP but marked as PAWN. Link: https://github.com/ItangSanjana/addressfield_id/search?l=pawn

**Comment by pchaigno** [comment]
You can fix this with [Linguist overrides](https://github.com/github/linguist#overrides).

@arfon Maybe we should have a simple heuristic for PHP such as:
```ruby
disambiguate ".inc" do |data|
  if data.include?("<?php")
    Language["PHP"]
  end
end
```

**Comment by ItangSanjana** [comment]
Thanks!

**Comment by arfon** [comment]
> @arfon Maybe we should have a simple heuristic for PHP such as:

Yeah, not a bad idea. This isn't the first time this has happened...

**Comment by arfon** [comment]
Fixed in #2980 - this will be live on GitHub next week.
